### PR TITLE
updated experiences model

### PR DESCRIPTION
**Pull request by rothquel**
Added the missing columns to our experiences model

### CODE REVIEW

File: app/controllers/experiences_controller.rb [hunk 1]
```diff
@@ -41,6 +41,6 @@ def set_experience
   end
 
   def experience_params
-    params.require(:experience).permit(:grandma_name, :activity_name, :availability, :description, :images, :price, :latitude, :longitude)
+    # params.require(:experience).permit(:grandma_name, :activity_name, :availability, :description, :images, :price, :latitude, :longitude)
```
Comment (anaisindia1996) [inline app/controllers/experiences_controller.rb:44]: Uncomment this line

File: db/migrate/20220815205138_add_fields_to_experience_model.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class AddFieldsToExperienceModel < ActiveRecord::Migration[7.0]
+  def change
+    add_column :experiences, :grandma_name, :string
+    add_column :experiences, :activity_name, :string
+    add_column :experiences, :availability, :string
+    add_column :experiences, :description, :text
+    add_column :experiences, :images, :string
+    add_column :experiences, :price, :integer
+    add_column :experiences, :latitude, :string
+    add_column :experiences, :longitude, :string
```
Comment (anaisindia1996) [inline db/migrate/20220815205138_add_fields_to_experience_model.rb:10]: Lets add location, :string so we can name the neighbourhood

File: app/controllers/experiences_controller.rb (hunk 1 above)
Comment (rothquel) [reply app/controllers/experiences_controller.rb:44]: Ouch

File: db/migrate/20220815205138_add_fields_to_experience_model.rb (hunk 2 above)
Comment (rothquel) [reply db/migrate/20220815205138_add_fields_to_experience_model.rb:10]: OK done, reuploading

**Review by anaisindia1996 — APPROVED** [review]
great!

**Pull request merged**